spec/models/like_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  test_user = User.create(name: 'Yankee', bio: 'Test fro NIgeria')
+  test_post = Post.create(author: test_user, title: 'test post', text: 'this is my test post')
```

Comment:
- [ ] Your like test is failing because you have failed to provide the correct attribute for creation of User and Post. You can do this instead:

  ```
  test_user = User.create(name: 'Yankee', bio: 'Test fro NIgeria', posts_counter: 0)
  test_post = Post.create(author: test_user, title: 'test post', text: 'this is my test post', comments_counter: 0, likes_counter: 0)
  ```